com.cubrid.common.ui/src/com/cubrid/common/ui/common/preference/JdbcManageComposite.java

```diff
@@ -139,7 +140,32 @@ public void save() {
 	private void createJdbcTableGroup(Composite composite) {
 		final String[] columnNameArr = new String[]{
 				Messages.tblColDriverVersion, Messages.tblColJarPath };
+		
 		TableViewerSorter sorter = new TableViewerSorter();
+		sorter.setColumnComparator(0, new Comparator<Object>(){
```

Comment:
Basically, we didn't have fixed coding style for own purpose, instead we are using eclipse default coding style except line width setting. Just for your reference...

![sj](https://cloud.githubusercontent.com/assets/1791415/20932470/170d923c-bc18-11e6-94e7-6750d81345c0.png)